modules/installation/pages/general-requirements.adoc

```diff
@@ -25,19 +25,18 @@ It is also not possible to use {slsa} for SAP as a base for {productname} for a
 [[install.scc-register]]
 == Obtaining your SUSE Customer Center Credentials
 
-You will need to create an account with {scc} before you install {sls} and {productname}.
-To obtain your {scc} credentials:
+Create an account with {scc} before installation of {sls} and {productname}.
```

Comment:
```suggestion
Create an account with {scc} before installation of {sls} and {productname}.
```

Why did you make this passive? The original worked better.

Reply:
Woops must have been a little tired thanks!